src/identity/IdentityPage.tsx

```diff
@@ -61,8 +63,9 @@ export const IdentityPage = () => {
     return updateUser({ ...user!, profile });
   }
 
-  function createAddress(address: Address) {
-    return updateUser({ ...user!, address });
+  async function createAddress(address: Address) {
+    await updateUser({ ...user!, address });
+    return history.push('/how-it-works');
```

Comment:
Think i wouldn't merge this into live while we have placeholder content there - we can merge the code but let's not yet add the redirect, would like live to be demoable at any moment

Reply:
Definitely, wasn't the plan to merge before we have the copy 👍